### DISCUSSION TITLE
Question about Updates

### DISCUSSION THREAD

**Discussion by trek-e** [mbround18/valheim-docker#369] [Q&A] [answered]
We have docker image running and it is doing great.   Recently, there was an update to Valheim and it didn't apply until I restarted the container.

I was curious what the update process for the server was.  I have Watchtower running (https://github.com/containrrr/watchtower) and it has updated the container fine, but it appears a game update came and didn't apply, and I was wondering what the update process for the game server itself was and if I missed something.

**Comment by mbround18** [comment, marked as answer]
If you have auto updates on, it:
- Checks if players are online
- If no, then checks if there is an update
- if yes, stops the valheim server
- Triggers a steam update
- Starts the server back up 

If you are not using auto updates, 
You can update the container by using the `FORCE_INSTALL` to trigger the update on container reboot
